src/combinatorics/configuration.lean

```diff
@@ -395,6 +395,30 @@ variables (P) {L}
 lemma point_count_eq [projective_plane P L] (l : L) : point_count P l = order P L + 1 :=
 (line_count_eq (dual P) l).trans (congr_arg (λ n, n + 1) (dual.order P L))
 
+variables (P L)
+
+lemma one_lt_order [projective_plane P L] : 1 < order P L :=
+begin
+  obtain ⟨p₁, p₂, p₃, l₁, l₂, l₃, -, -, h₂₁, h₂₂, h₂₃, h₃₁, h₃₂, h₃₃⟩ := @exists_config P L _ _,
+  classical,
+  rw [←add_lt_add_iff_right, ←point_count_eq, point_count, nat.card_eq_fintype_card],
+  exact let h_ne : l₁ ≠ l₂ := λ h, h₂₁ ((congr_arg _ h).mpr h₂₂) in let h := mk_point_ax h_ne in
```

Comment:
This feels less weird:
```suggestion
  rw [←add_lt_add_iff_right, ←point_count_eq _ l₂, point_count, nat.card_eq_fintype_card, ←bit0,
    fintype.two_lt_card_iff],
  have h_ne : l₁ ≠ l₂ := λ h, h₂₁ ((congr_arg _ h).mpr h₂₂),
  have h := mk_point_ax h_ne,
  exact ⟨⟨p₂, h₂₂⟩, ⟨p₃, h₃₂⟩, ⟨mk_point h_ne, h.2⟩,
```
I get the feeling you might be missing a helper lemma somewhere about `ne` and `mem`.